documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -1712,9 +1712,15 @@ For example, if you set `max.queue.size=1000`, and `max.queue.size.in.bytes=5000
 
 |[[vitess-property-skipped-operations]]<<vitess-property-skipped-operations, `+skipped.operations+`>>
 |`t`
-|A comma-separated list of operation types that will be skipped during streaming.
-The operations include: `c` for inserts/create, `u` for updates, `d` for deletes, `t` for truncates, and `none` to not skip any operations.
-By default, truncate operations are skipped (not emitted by this connector).
+|A comma-separated list of the operation types that you want the connector to skip during streaming.
+You can configure the connector to skip the following types of operations:
+
+* `c` (insert/create)
+* `u` (update)
+* `d` (delete)
+* `t` (truncate)
+
+Set the value to `none` if you do not want the connector to skip any operations.
```

Comment:
@twthorn With the DDL parser in place, does Vitess emit truncate event or no?

Reply:
It emits the truncate events to the ddl topics, but we do not currently emit them within the data change topic (as I believe some other connectors do, eg mysql)